tivars/models/__init__.py

```diff
@@ -2,7 +2,7 @@ class TIModel:
     def __init__(self, name: str, flags: int, signature: str):
```

Comment:
I'm concerned that this signature field here might be conflated with a cryptographic signature or something else; I think it's more appropriate to break away from what tivarslibcpp does here and call it `file_magic` or just `magic`, see https://en.m.wikipedia.org/wiki/Magic_number_(programming)#In_files

Reply:
"Magic" feels rather non-descriptive, even owing to the term's precedence, but I guess it isn't potentially actively confusing

Reply:
yeah, it's not the greatest, but it is an arbitrary number ("magic number") that has meaning

Reply:
delete this

Reply:
Yeah, I'm on board with this, its just a matter of deciding between `magic` or `file_magic` or some secret third thing